src/CONST.js

```diff
@@ -47,6 +47,7 @@ const CONST = {
         TYPE: {
             CHAT: 'chat',
             IOU: 'iou',
+            ADDCOMMENT: 'ADDCOMMENT',
```

Comment:
You've created `REPORT.TYPE.ADDCOMMENT` here, but we want `REPORT.ACTION.TYPE.ADDCOMMENT`

Reply:
sorry, fixed